Sources/Features/Metrics/Model/Event.swift

```diff
@@ -0,0 +1,21 @@
+import Foundation
+
+protocol Event: Encodable {
+
+    var timestamp: Int64 { get set }
+    var attributes: DecodableAttributes { get set }
+}
+
+extension Event {
+    func initialTimestamp() -> Int64 {
+        return Date().currentTimeSeconds()
+    }
```

Comment:
Just a suggestion to make more testable: 

```suggestion
    static func initialTimestamp(_ date: Date = Date()) -> Int64 {
        return date.currentTimeSeconds()
    }
```

Reply:
Good idea, thank you!